reference/docs-conceptual/dev-cross-plat/vscode/using-vscode-for-debugging-compiled-cmdlets.md

```diff
@@ -0,0 +1,204 @@
+---
+ms.date:  10/19/2020
+keywords:  powershell,cmdlet,debug
+title:  Using Visual Studio Code for debugging compiled cmdlets
+description: How to set a build task and launch configuration for a PSModule project in .NET Core
+---
+# Using Visual Studio Code for debugging compiled cmdlets
+
+This guide shows you how to interactively debug C# source code for a compiled PowerShell module
+using Visual Studio Code and the C# extension.
+
+Some familiarity with the Visual Studio Code debugger is assumed.
+
+- For a general introduction to the Visual Studio Code debugger, see
+  [Debugging in Visual Studio Code][].
+
+- For examples of debugging PowerShell script files and modules, see
+  [Using Visual Studio Code for remote editing and debugging][].
+
+This guide follows on from the [Writing Portable Modules][] guide.
```

Comment:
"Follows on" feels weird.  Can we rephrase this?

Reply:
Fixed